envisage/tests/test_api.py

```diff
@@ -0,0 +1,27 @@
+# (C) Copyright 2007-2023 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+
+import unittest
```

Comment:
Please follow the usual policy for import groups. On most ETS projects, we use the following division:

- Standard library imports
- Imports from third parties (e.g., NumPy, h5py, lxml, ...)
- Imports from other Enthought packages (e.g., traits, pyface, ...)
- Local imports

with a blank line between each group. Here `import unittest` falls into the "Standard library imports" group, so we need a blank line after the `import unittest`.

Reply:
Fixed; thank you!